src/headless/cli.ts

```diff
@@ -0,0 +1,86 @@
+import { Command } from 'commander';
+import { CLIOptions } from '../types/CLIOptions';
+import { DataLoader } from './dataLoader';
+import { runShadingSceneHeadlessChrome } from './headlessBrowser';
+
+// Turn file names into array
+function fileArray(input: string | string[] | undefined): string[] {
+  if (!input) return [];
+  return Array.isArray(input) ? input : [input];
+}
+
+/**
+ * The CLI program that defines the input and output parameters, provides explanations, and converts the
+ * parameters into the appropriate formats for the headless program.
+ */
+export async function main(argv: string[]) {
```

Comment:
I added `export` here, so that the function is taken into account by typedoc. I hope this does not change any functionality. If you think this should not be exported, I can have a look at typedoc docs to see if there are other ways to achieve this.